RiotSwiftUI/Modules/Room/VoiceBroadcastPlayback/MatrixSDK/VoiceBroadcastPlaybackViewModel.swift

```diff
@@ -333,6 +341,9 @@ extension VoiceBroadcastPlaybackViewModel: VoiceBroadcastAggregatorDelegate {
     }
     
     func voiceBroadcastAggregatorDidEndLoading(_ aggregator: VoiceBroadcastAggregator) {
+        if state.playbackState == .buffering {
```

Comment:
@phlniji please remove this block, the process will be launched on voiceBroadcastAggregatorDidUpdateData

Reply:
Updated in last commit.